pipeline/cloud/pipelines.py

```diff
@@ -241,7 +243,7 @@ def map_pipeline_mp(array: list, graph_id: str, *, pool_size=8):
 def stream_pipeline(
     pipeline_id_or_pointer: str,
     *data,
-) -> t.Iterator[RunOutput]:
+) -> t.Iterator[t.Any]:
```

Comment:
```suggestion
) -> t.Iterator[t.Union[RunOutput, str]]:
```

Reply:
If I make the changes below the return type could either be a `RunOutput` or a `RunError`. The means that a user calling `stream_pipeline` will need to do something like the following:

```python
for output in stream_pipeline(pointer, *inputs):
    if isinstance(output, RunOutput):
        print(output.value)
    elif isinstance(output, RunError):
        print(output.traceback or output.exception)
```

which adds more work for the user and seems a little messy. What do you think?

Reply:
I actually think a better thing to do would be to raise an exception?

Reply:
yeah good idea!